config/initializers/okcomputer.rb

```diff
@@ -11,4 +11,14 @@ def check
   end
 end
 
+# OKComputer check that checks if we have a connection to symws
```

Comment:
This checks the connection to Symphony database server, not web services. How often does this check occur? When app is deployed?

Reply:
Nagios checks this endpoint (every 5 minutes?) as part of its normal health checks.

Reply:
Thanks!